docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+networks: 
+  todo-net:
+    driver: bridge
+services:
+  todoapp:
+    image: todoapp:2.0.0
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp
+    ports:
+      - "8080:8080"
+    environment:
+      - ENV=local
```

Comment:
This App didn't use the variable `ENV` so it is useless

Reply:
fixed